brat/Brat/Parser.hs

```diff
@@ -487,12 +487,17 @@ expr' p = choice $ (try . getParser <$> enumFrom p) ++ [atomExpr]
         Nothing -> unWC expr
         Just rest -> FJuxt expr rest
 
+  fanout = square (FFanOut <$ match Slash <* match Backslash)
+  fanin = square (FFanIn <$ match Backslash <* match Slash)
```

Comment:
Does this imply that we can also write `[ /\ ]`?

Have you considered lexing whole tokens for fanin and fanout instead of decomposing into slash and backshlash?

Reply:
Yes, and I'm not sure whether to consider that a good thing or a bad thing 😛 lexing the patterns should work just as well

Reply:
Yeah, I don't have a strong opinion either 😅 I'm fine with leaving it as is if you're happy with it